src/test/java/karuna/karuna_backend/post/domain/PostServiceTest.java

```diff
@@ -18,12 +17,13 @@ class PostServiceTest {
     void shouldCreatePost() {
         //when: When create post, returns created post
         PostCreateDto postCreateDto = new PostCreateDto(Constants.BODY);
-        Principal principal = () -> Constants.NAME;
         PostDto postDto = postService.createPost(postCreateDto);
+        String username = AuthenticationUtil.getUsername();
         //then: Check if  database give id and correctly save data
         assertEquals(1L, postDto.id());
         assertNotNull(postDto.createdAt());
         assertEquals(Constants.BODY, postDto.body());
+        assertEquals(username, postDto.author());
```

Comment:
This test case give false success result, because u check here if null is equal null, its not possible, to inject user to security? @BarSwi what u think? we should leavt it like this, or add user to "context"

Reply:
Change username, to Constants,USERNAME